.github/workflows/build_playground_backend.yml

```diff
@@ -85,3 +85,9 @@ jobs:
           GOOGLE_APPLICATION_CREDENTIALS: /tmp/gcp_access.json
         run: ./gradlew playground:terraform:deployBackend -Pdocker-tag=${{env.DOCKERTAG}} -Pproject_id=${{ secrets.GCP_PLAYGROUND_PROJECT_ID }} -Pproject_environment='beta' -Pdocker-repository-root='${{ secrets.PLAYGROUND_REGISTRY_NAME}}/${{ secrets.GCP_PLAYGROUND_PROJECT_ID }}/playground-repository'
         if: startsWith(github.ref, 'refs/tags/')
+      - name: DB Index creation
+        run: |
+         gcloud auth activate-service-account --project=${{ secrets.GCP_PLAYGROUND_PROJECT_ID }} --key-file=/tmp/gcp_access.json
```

Comment:
This is not a blocker on the PR, but I want to ask, there's no way to use service account impersonation instead of activating a service account via a downloaded key?

Reply:
I got following error even if I try to use "--impersonate-service-account SERVICE_ACCOUNT":
WARNING: This command is using service account impersonation. All API calls will be executed as [ms1-demo-svc@***.iam.gserviceaccount.com].
WARNING: Could not open the configuration file: [/home/runner/.config/gcloud/configurations/config_default].
ERROR: (gcloud.app.deploy) You do not currently have an active account selected.